index.html

```diff
@@ -274,7 +275,40 @@ <h2 class="connect">
                 <hr />
             </div>
         </section>
-
+        <!-- Modal -->
```

Comment:
Your modal looks good and works well 💯.  However, it does not follow the Figma design closely enough:

Your modal:

![image](https://user-images.githubusercontent.com/57421823/151008741-8f1b89a0-514a-4cb7-b81a-48e4eb244b7a.png)

![image](https://user-images.githubusercontent.com/57421823/151012483-d4df0a44-7638-4f43-80d5-689bd8644f1f.png)

This is the Figma modal:

![image](https://user-images.githubusercontent.com/57421823/151012723-65bad042-c172-4530-8a2e-1343a9e7e616.png)


Please, kindly:

- [ ] Update the modal font stylings, so the sizes, weights, line-heights, etc matches the Figma.
- [ ] Update the modal main window so it has slightly rounded corners, like in the Figma.
- [ ] Update the modal main featured image, so it has also slight rounded corners, like in the Figma.
- [ ] The "See live" and "See source" buttons should be aligned horizontally like in the Figma template, not vertically, please kindly do so. 
- [ ] The line above the "See live" button, please make it semi-transparent and extend it so it spans about 90% of the modal, horizontally.  Please check the Figma to ensure this line looks the same. 💪 

_Note:  The example that is given in the project requirements, in which the modal buttons are aligned vertically, is just an example using template 5.  You are using template 1, which uses horizontal buttons as seen above._

If you implement this changes, your portfolio project modals will look striking!! 💪 💪